docs/getting-started/index.md

```diff
@@ -1,106 +1,97 @@
+---
+title: Getting started
```

Comment:
Why is this necessary, afaik it takes the title from the first anchor tag

Reply:
I though there's a convention that page headers use "Title Case" and the sidebar items use "Sentence case". Looks like it's not the case 😄 

I think I'll submit another PR later to make it consistent across docs: https://github.com/chartjs/Chart.js/issues/10819